packages/components/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "dist/alpheios-components.min.js",
   "scripts": {
     "test": "jest --coverage  && shx cat ./coverage/lcov.info | coveralls",
-    "test-no-coverage": "jest tests/lib && jest tests/vue",
-    "test-ci": "npm run test-lib && npm run test-components",
+    "test-no-coverage": "jest tests/lib && jest tests/vue  && jest tests/inflections",
+    "test-ci": "npm run test-lib && npm run test-components ",
```

Comment:
so we're testing only vue components when travis runs? Is this what we want?

Reply:
No - this way we test all tests
tests/lib - not vue components
tests/vue - vue components
tests/inflections - full inflection browser tests